src/Schema/Directives/DeleteDirective.php

```diff
@@ -45,9 +63,11 @@ function ($root, array $args) {
                 if ($this->directiveArgValue('globalId', false)) {
                     // At this point we know the type is at least wrapped in a NonNull type, so we go one deeper
                     if ($argumentDefinition->type->type->kind === NodeKind::LIST_TYPE) {
-                        $idOrIds = array_map([GlobalId::class, 'decodeId'], $idOrIds);
+                        $idOrIds = array_map(function ($id) {
```

Comment:
Let's break this into multiple lines so the arguments align.

Reply:
Not sure what you mean here @spawnia ?